From 157b6fdb13eb8bc60eb67ce3abc57c8ae5737a05 Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Tue, 14 Feb 2012 11:13:56 +0100
Subject: [PATCH 31/99] raw-posix: raw_pwrite comment fixup

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1329218101-24213-32-git-send-email-kwolf@redhat.com>
Patchwork-id: 37223
O-Subject: [RHEL-6.3 qemu-kvm PATCH v2 31/96] raw-posix: raw_pwrite comment fixup
Bugzilla: 783950
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Christoph Hellwig <hch@lst.de>

Bugzilla: 783950

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Kevin Wolf <kwolf@redhat.com>
(cherry picked from commit 11a3cb8159278f7452b5bec8b9e17292a3ef53c3)
---
 block/raw-posix.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/raw-posix.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/block/raw-posix.c b/block/raw-posix.c
index 735fe49..c42d31b 100644
--- a/block/raw-posix.c
+++ b/block/raw-posix.c
@@ -504,7 +504,7 @@ static int raw_pwrite(BlockDriverState *bs, int64_t offset,
                 count -= ret;
                 sum += ret;
             }
-            /* here, count < 512 because (count & ~sector_mask) == 0 */
+            /* here, count < sector_size because (count & ~sector_mask) == 0 */
             if (count) {
                 ret = raw_pread_aligned(bs, offset, s->aligned_buf,
                                      bs->buffer_alignment);
-- 
1.7.7.5

